From 6246a99df73663c554bffbe7d83b712f50c66df5 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Wed, 7 Mar 2018 08:27:54 +0800
Subject: [PATCH] Revert "FROMLIST: mmc: dw_mmc: Fix the DTO timeout overflow
 calculation for 32-bit systems"

This reverts commit 9c8f6bbf418b3f7e7425f116a84e603aebbb1cab.

Change-Id: I526d0748a998520ac3e65098c4d4cb4aa9ef4545
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 drivers/mmc/host/dw_mmc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mmc/host/dw_mmc.c b/drivers/mmc/host/dw_mmc.c
index 6e3ea0856e7b..8730e2e8bc77 100644
--- a/drivers/mmc/host/dw_mmc.c
+++ b/drivers/mmc/host/dw_mmc.c
@@ -1768,8 +1768,8 @@ static void dw_mci_set_drto(struct dw_mci *host)
 	drto_div = (mci_readl(host, CLKDIV) & 0xff) * 2;
 	if (drto_div == 0)
 		drto_div = 1;
-	drto_ms = DIV_ROUND_UP_ULL(MSEC_PER_SEC * drto_clks * drto_div,
-				   host->bus_hz);
+	drto_ms = DIV_ROUND_UP(MSEC_PER_SEC * drto_clks * drto_div,
+			       host->bus_hz);
 
 	/* add a bit spare time */
 	drto_ms += 10;
-- 
2.35.3

